### DISCUSSION TITLE
Tell the user to use the workaround for explicit content if they're in a restricted country.

### DISCUSSION THREAD

**Discussion by xBiei** [ciderapp/Cider-2#398] [Feature Requests]
Today I tried to listen to my music on Cider but it told me that I can't listen to explicit music because it's restricted in my storefront or something like that.

So I did some research and found a way to workaround this without breaking stuff.

Steps for the workaround:
- Go to iTunes Store app on your iDevice.
- Select an explicit song.
- Click the price button as if you're purchasing the song. (You're not gonna buy it dw)
- You'll see an alert saying "Explicit content blah blah".
- Click allow.

If you get a popup saying you need to allow explicit content you can do that through:
- Settings.
- Screen Time.
- Content & Privacy Restrictions.
- Content Restrictions.
- Music, ...
- Choose Explicit.

Adding this to the app will surely make lives easier because it's a headache and stupid.

**Comment by lilpopsmoke** [comment]
still not working...

**Reply by xBiei** [reply]
Did you restart Cider after doing this?
EDIT: Also, did you see the Explicit content popup on your iDevice and clicked Allow?

**Comment by lilpopsmoke** [comment]
On Tue, 14 Nov 2023 at 12:06 PM _xB ***@***.***> wrote:

> Did you restart Cider after doing this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ciderapp/Cider-2/discussions/398#discussioncomment-7563744>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A55RJBSMBW5M2E6R77ZBDT3YEM7CTAVCNFSM6AAAAAA5TMNDWGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNRTG42DI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
Yes i did then I uninstalled the app then installed it again & still the
same issue happening, i’m sorry but i paid for this to work normally!

**Reply by xBiei** [reply]
Well, it's supposed to work out of the box but due to country restrictions this happens.
Ok try to play explicit music on ur iDevice first and check if it's working.
If it does work on your iDevice then ask for a refund, if it doesn't then there's something wrong with the process.
If you did everything right, then the grayed buy button in explicit content on itunes store should look normal buy button.

**Reply by xBiei** [reply]
You should also check your `Content & Privacy Restrictions` in screentime in your settings and make sure explicit content is enabled
